Load record types sdk functions added

sgaddala-ks opened the pull request.

sk-keeper commented on KeeperSdk/vault/RecordTypeCommands.cs:
```diff
@@ -205,13 +243,92 @@ public void checkAdminAccess()
                 }
             }
+
+            public List<InputRecordType> ValidateRecordTypeFile(string filePath)
```
This method does not belong to SDK space.
What is the usage of this method inside SDK?

sk-keeper commented on KeeperSdk/vault/RecordTypeCommands.cs:
```diff
@@ -100,6 +101,43 @@ public async Task<string> DeleteRecordTypeAsync(string recordTypeId)
             }
         }
 
+        public async Task<List<string>> LoadRecordTypesAsync(string filePath)
```
This method seems to belong to the Commander not SDK.
Is it used inside the SDK?

sk-keeper commented:
Can you move `RecordTypeCommands.cs` file to the Commander project please?
The file that uses `Console` does not belong to SDK. 
The Keeper SDK is already used by the WPF projects. 
Keeper SDK should not have any UI assumption heither CLI nor GUI.

sgaddala-ks commented:
I have removed all UI/CLI elements I have used earlier.
Renamed `RecordTypeCommands.cs` to `RecordTypeExtensions.cs` following sdk pattern.
moved the implementation to Commander
I am also working on updating the same changes to [PowerShell Command in next PR](https://github.com/Keeper-Security/keeper-sdk-dotnet/pull/181)
